pyproject.toml

```diff
@@ -45,6 +45,11 @@ dependencies = [
     "pytest-testconfig",
     "python-simple-logger",
     "pyyaml",
+    "syrupy>=4.0.1",
+    "caikit-nlp-client>=0.0.8",
+    "requests>=2.31.0",
+    "aiohttp>=3.8.5",
+    "pytest-asyncio>=0.21.1",
```

Comment:
`grpcio` is missing

Reply:
grpcio will be installed as part of caikit-nlp-library no need to mention that. Also i normally add the version because i don't want pip  backtracking. but i can remove those